iroh/src/commands/sync.rs

```diff
@@ -98,6 +117,49 @@ pub enum DocCommands {
         /// Optional key prefix (parsed as UTF-8 string)
         prefix: Option<String>,
     },
+    /// Import data into a document
+    Import {
+        /// Document to operate on.
+        ///
+        /// Required unless the document is set through the IROH_DOC environment variable.
+        /// Within the Iroh console, the active document can also be set with `doc switch`.
+        #[clap(short, long)]
+        doc: Option<NamespaceId>,
+        /// Author of the entry.
+        ///
+        /// Required unless the author is set through the IROH_AUTHOR environment variable.
+        /// Within the Iroh console, the active author can also be set with `author switch`.
+        #[clap(short, long)]
+        author: Option<AuthorId>,
+        /// Prefix to add to imported entries (parsed as UTF-8 string). Defaults to no prefix
+        #[clap(long)]
+        prefix: Option<String>,
+        /// Path to a local file or directory to import
+        ///
+        /// Pathnames will be used as the document key
+        #[clap(long)]
+        path: String,
+        /// If true, don't copy the file into iroh, reference the existing file instead
+        ///
+        /// Moving a file imported with `in-place` will result in data corruption
+        #[clap(short, long)]
+        in_place: bool,
+    },
+    /// Export data from a document
+    Export {
+        /// Document to operate on.
+        ///
+        /// Required unless the document is set through the IROH_DOC environment variable.
+        /// Within the Iroh console, the active document can also be set with `doc switch`.
+        #[clap(short, long)]
+        doc: Option<NamespaceId>,
+        /// Key to the entry (parsed as UTF-8 string)
+        #[clap(short, long)]
+        key: String,
```

Comment:
I don't think this needs to happen in this PR. But we can import entire directories, but export only individual blobs, correct? It would be nicer if we could also export entire directories. We could maybe make this an option, and if it is None export the entire doc.

Reply:
yea i had the same thought: https://github.com/n0-computer/iroh/pull/1504#discussion_r1340695276

The conclusion was, let's wait for someone to ask for it.